code/datums/uplink_item.dm

```diff
@@ -225,9 +216,17 @@ var/list/uplink_items = list()
 		"Psychiatrist",
 		"Chemist",
 		"Paramedic",
+		"Coroner",
 		"Virologist",
-		"Brig Physician"
-	)
+		"Brig Physician") // in the very rare instance that this is possible
```

Comment:
Brig Phys should be removed from this; he got a loyalty implant an antag exclusion long ago; when this was added, Brig Phys could be antag.